crypto/src/hash/rpo/digest.rs

```diff
@@ -0,0 +1,112 @@
+use super::DIGEST_SIZE;
+use crate::{Digest, Felt, StarkField};
+use core::slice;
+use winterfell::{ByteReader, ByteWriter, Deserializable, DeserializationError, Serializable};
```

Comment:
when I change this line from `winterfell` to `winter_utils` I get:
the trait bound `RpoDigest256: winterfell::Deserializable` is not satisfied

Reply:
hmm - could be because of version mismatch. are you able to remove `winterfell` dependency completely and use only individual dependencies from crates.io?

Reply:
Fixed with the recent version update.